src/encoded/visualization.py

```diff
@@ -1144,3 +1198,91 @@ def add_zoom_lock_if_needed(view_config, view, scales_and_center_k):
         # Copy the initialXDomain and initialYDomain
         view["initialXDomain"] = view_config["views"][0]["initialXDomain"] or view["initialXDomain"]
         view["initialYDomain"] = view_config["views"][0]["initialYDomain"] or view["initialYDomain"]
+    return True
+
+def remove_left_side_if_all_1D(new_views):
+    """ If the view config has no 2D files, then remove the left side from the view config.
+
+    Args:
+        new_views(list): The views that will make the new HiGlass view config. May be modified.
+
+    Returns:
+        A boolean value indicating success.
+    """
+
+    # Search all views' central contents for any 2D files.
+    for view in new_views:
+        for center_track in view["tracks"]["center"]:
+            if "contents" not in center_track:
+                continue
+
+            # If 2D files are found, we shouldn't remove any tracks.
+            if any([ t for t in center_track["contents"] if t["type"] in ("heatmap", "2d-chromosome-grid")]):
+                return True
```

Comment:
Should this be 'False'?

Reply:
Maybe a boolean response is too vague. True meant "no errors found" instead of "I definitely removed the left tracks. I can change it without trouble, though.

Reply:
Ah kk makes sense. Might be cool to have it return 'False' and if any error then let it throw an exception or something in case we wanna know in future if function did/didn't delete a track. Idk, have no real preference.